### ISSUE TITLE
There is a Bug in “/_layouts/home.html”??

### ISSUE THREAD

**Issue by CS2015YWB** [github/personal-website#99]
### **/personal-website/_layouts/home.html**
```
<div class="d-md-flex height-full {% unless site.style == 'dark' %}border-md-bottom{% endunless %}">
```
**the `height-full`  will cause layout confusion at the bottom when we view in desktop.
just like this:**
![image](https://wx2.sinaimg.cn/mw690/006aCS7gly1g12f0i84ryj311y0szwhr.jpg)

**when I remove `height-full`, it is okay!**
![image](https://wx4.sinaimg.cn/mw690/006aCS7gly1g12f3fpn3nj311y0u4ade.jpg)

**Comment by Sunyelw** [comment]
sikuoyi !

**Comment by CS2015YWB** [comment]
Besides，there is a same bug in"/personal-website/_layouts/post.html"

**Comment by CS2015YWB** [comment]
Besides，there is a same bug in"/personal-website/_layouts/post.html" when we view the posts!

**Comment by brandonrosage** [comment]
We merged #97 today, which should address this bug. Thanks!
